app/_hub/kong-inc/kafka-upstream/how-to/_multiple-topics-and-acl.md

```diff
@@ -0,0 +1,82 @@
+---
+nav_title: Configure Multiple Producer Topics with Whitelist in Kafka Upstream Plugin
+title: Configure Multiple Producer Topics with Whitelist in Kafka Upstream Plugin
+minimum_version: 3.10.x
+---
+
+## Configure Multiple Producer Topics with Allowlist in Kafka Upstream Plugin
+
+
+Since {{site.base_gateway}} 3.10, the Kafka Upstream Plugin supports sending the message to multiple client-defined Kafka topics, by using a query parameter that contains a list of target topic names. To prevent client request from sending message to arbitrary topics, user can also define a topic allowlist that can be sent to.
```

Comment:
```suggestion
Starting in {{site.base_gateway}} 3.10, the Kafka Upstream plugin supports sending messages to multiple client-defined Kafka topics by using a query parameter that contains a list of target topic names. 
To prevent client requests from sending messages to arbitrary topics, you can also define a topic allowlist.
```